src/bindings/python/flux/job.py

```diff
@@ -260,7 +260,7 @@ def job_list(flux_handle, max_entries=0, attrs=[], userid=os.geteuid(), states=0
 
 def job_list_inactive(flux_handle, since, max_entries=1000, attrs=[]):
     payload = {
-        "timestamp": since,
+        "since": since,
```

Comment:
for commit message for this change, I might instead say "Rename the 'timestamp' payload field to 'since' in `job-info.list-inactive`, adjust appropriately in libjob/job.[ch] and rename --timestamp to --since in cmd/flux-job."  Or something like that.